src/buildstream_plugins/elements/autotools.yaml

```diff
@@ -17,6 +17,11 @@ variables:
   # Project-wide extra arguments to be passed to `configure`
   conf-global: ''
 
+  # This needs to be overridden for build-dir to work
```

Comment:
Two things here:
* The sentence *"This needs to be overridden for build-dir to work command-subdir is for monorepos to work"* doesn't read very well and appears to be developer facing technical detail. As this yaml is rendered in [user facing documentation](https://apache.github.io/buildstream-plugins/elements/autotools.html), comments here should be user facing, e.g.:
  * `conf-root`: The directory where the configure script will be generated and used
  * `command-subdir`: The subdirectory of `%{build-root}` where commands will be run
* I think this should be positioned near the `build-dir` as they seem functionally related

Reply:
Thanks. I will fix once I have confidence that this is sufficient to make build-dir work. I have MR open at https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/merge_requests/9438

Reply:
Looks like build-dir unsetting now works different than before ("." vs "" ) so I will need to spend some time fixing freedesktop-sdk. I will comment back here when I get a passing pipeline.

Reply:
Looks like the suggested way is to leak element name into Makefile as per https://github.com/apache/buildstream-plugins/blob/master/tests/elements/autotools/elements/amhelloconfroot.bst#L14 so I'll drop trying to get relative paths done.